skl2onnx/operator_converters/gaussian_process.py

```diff
@@ -40,6 +40,8 @@ def convert_gaussian_process_regressor(scope, operator, container):
     out = operator.outputs
     op = operator.raw_operator
     opv = container.target_opset
+    if opv is None:
+        raise RuntimeError("container.target_opset must be None")
```

Comment:
Did you want to say "container.target_opset must not be None"?

Reply:
fixed